common.js

```diff
@@ -41,6 +42,29 @@ function alphasort (a, b) {
   return a.localeCompare(b)
 }
 
+function ignoreMap (self, options) {
+  self.ignore = options.ignore || []
+
+  if (!Array.isArray(self.ignore))
+    self.ignore = [self.ignore]
+
+  if (self.ignore.length) {
+    self.ignore = self.ignore.map(function (pattern) {
+      var endsWithGlobStar = pattern.slice(-3) === '/**'
+      var endsWithStar = pattern.slice(-2) === '/*'
+      var m = { matcher: new Minimatch(pattern) }
+
+      //Using indexOf to match 'a' in case of patterns like a/**/**
+      if (endsWithGlobStar)
+        m.gmatcher = new Minimatch(pattern.slice(0, pattern.indexOf('/**')))
```

Comment:
This will be incorrect in the case of `a/**/b/**` where you want it to end up with `a/**/b`, and instead, it'll be `a`.